src/interfaces/chain.h

```diff
@@ -280,6 +282,12 @@ class ChainClient
 
     //! Shut down client.
     virtual void stop() = 0;
+
+    //! Set mock time.
+    virtual void setMockTime(int64_t time) = 0;
+
+    //! Return interfaces for accessing wallets (if any).
+    virtual std::vector<std::unique_ptr<Wallet>> getWallets() { return {}; }
```

Comment:
Any reason to have definition?

Reply:
I presume this is needed when there is more than one chain client that has wallets. Maybe the doc string could be clarified to say "of the wallets of all chain clients" or similar.